files-community/Files
# Feature: Added a Command Palette

**Resolved / Related Issues**
   Closes #2780 

**Validation**
How did you test these changes?
  1. Type ">" in the address bar or press Ctrl+Shift+P to open the command palette
  2. You can search commands by both description (translated) and command code.
  3. Select a suggestion or type a command code to invoke the command.

**Screenshots**
![image](https://github.com/files-community/Files/assets/66369541/61ab182d-9e9b-4dd4-8634-bd3f3438761f)
